.woodpecker/.latest.yml

```
@@ -4,7 +4,10 @@ steps:
     settings:
       repo: ${CI_REPO}
       tags: latest
-    secrets: [ docker_username, docker_password ]
+      username:
+        from_secret: docker_username
+      password:
+        from_secret: docker_password
 when:
-  branch: master
-  event: push
+  - event: push
+    branch: [development]
```

I would stick with `master` maybe? That's what the ci-templates repo uses as well.

```suggestion
    branch: [master]
```

---

Ah, I'm maybe too accustomed to OP using `latest` on the dev branch to deploy to dev and then tagged versions for QA and Prod :grin: but fair point, we can stick to how it was done until now.

---

I mean, if there is a good reason to deviate that's fine :+1: I just think this app isn't maintained  actively enough to even need a development branch :smile: